syntax = "proto3";

package teos;

message RegisterRequest {
  string user_id = 1;
}
message RegisterResponse {
  string user_id = 1;
  uint32 available_slots = 2;
  uint32 subscription_expiry = 3;
  string subscription_signature = 4;
}